vortexasdk/endpoints/canal_transit.py

```diff
@@ -0,0 +1,185 @@
+from datetime import datetime, timedelta
```

Comment:
Binder links will be added once we merge a PR, I want to make sure we'll get proper URLs and Binder is reading from `master` branch to generate proper files and links